### PR TITLE
feat(ScrollMenu): increase max height

### PR DESCRIPTION
**Pull request by pudek357**
[FX-2349]

### Description
As confirmed (please check ticket), Picasso **Autocomplete** should have a bigger max height, so the scrollbar is shown only for a number of items > 8.

### How to test
- go to: https://picasso.toptal.net/FX-2349-increase-max-height-for-scroll-menu/?path=/story/picasso-readme--readme
- check following components:
  - `Page.Autocomplete`
  - `Autocomplete`

### Screenshots
**Page.Autocomplete (no scrollbar): 4 items on the left, 8 items on the right**
| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
|  |  |

**Page.Autocomplete: 10 items**
| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
|  |  |

**Autocomplete**
| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
|  |  |

### Review

- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure that tests pass by running `yarn test`
- [x] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and
1. You have no extra requests.
2. You have optional requests.
   1. Add `nit: ` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because
1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:
1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2349]: https://toptal-core.atlassian.net/browse/FX-2349?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pudek357** [comment]
@toptal-bot run package:alpha-release

**Comment by pudek357** [comment]
@toptal-bot run all

File: packages/picasso/src/PageAutocomplete/story/Default.example.tsx [hunk 1]
```diff
@@ -21,7 +23,7 @@ const renderOption = ({ text }: Item) => (
 
 const Example = () => (
   <>
-    <div style={{ height: '4.5rem' }}>
+    <div style={{ height: '18.75rem' }}>
```
Comment (pudek357) [inline packages/picasso/src/PageAutocomplete/story/Default.example.tsx:26]: Increased height, to be able to show properly more items in `Autocomplete`:
image

**Review by OleksandrNechai — APPROVED** [review]
LGTM codewise. But I am not sure about the BASE team's approval. Please do not merge before you have approval from them.

**Comment by pudek357** [comment]
BASE conversation: https://toptal-core.slack.com/archives/CCC3GP6CC/p1639130714306100

**Review by hweeTan — APPROVED** [review]
Tested on temploy. LGTM

**Pull request merged**